person-service/src/main/kotlin/com/thehuginn/enums/PersonRole.kt

```diff
@@ -0,0 +1,7 @@
+package com.thehuginn.enums
+
+enum class PersonRole {
```

Comment:
PersonType. Pouzivam ptm kludne bud klavesove skratky, alebo right click  -> refactor -> rename alebo tak nejak. To iste aj pre premenne